recipe/meta.yaml

```diff
@@ -28,6 +28,7 @@ requirements:
     - blas 1.1 {{ variant }}
     - curl
     - pcre
+    - lapack
```

Comment:
Replace this with `openblas 0.2.19|0.2.19.*`. `openblas` is an optimized blas and lapack library and is better than `lapack` which is the reference implementation with no optimization

Reply:
But should I keep `blas 1.1`? 

Are you sure `openblas` has LAPACK? I can't find any mention of it on their website.

Reply:
`blas 1.1` is there to have one implementation of blas per env. @ocefpaf, @jakirkham can explain better than me.

Yes, I'm sure `openblas` has lapack

Reply:
Here are the [docs]( https://conda-forge.org/docs/blas.html ). Yes `openblas` implements both the BLAS and LAPACK interfaces within the same library. So it serves both purposes.

Reply:
OK, thanks for the explanation

Reply:
Sorry I realize that doc mention was super terse. If you have any questions, please feel free to ask.

Reply:
@jakirkham not at al. I didn't think to look in the conda-forge docs and it's good to know that the information is there. Reply time for all conda-forge is impressively low!